app/src/main/java/org/fossasia/openevent/general/data/Network.kt

```diff
@@ -0,0 +1,17 @@
+package org.fossasia.openevent.general.data
+
+import android.content.Context
+import android.net.ConnectivityManager
+import org.fossasia.openevent.general.OpenEventGeneral
+
+class Network {
+    private val connectivityManager by lazy { context.getSystemService(Context.CONNECTIVITY_SERVICE) as? ConnectivityManager }
```

Comment:
Should be below context, and move the body to next line

Reply:
Updated PR :+1: